tests/test_main.py

```diff
@@ -576,3 +576,15 @@ class MyModel(BaseModel):
     assert "json" in dir(m)
     assert "attribute_a" in dir(m)
     assert "attribute_b" in dir(m)
+
+
+def test_dict_with_extra_keys():
+    class MyModel(BaseModel):
+        a: str = Schema(None, alias='alias_a')
+
+        class Config(BaseConfig):
```

Comment:
you don't need to inherit from `BaseConfig` here.

Reply:
Thanks! @samuelcolvin